mobile/native/src/api.rs

```diff
@@ -286,11 +286,13 @@ pub fn get_unused_address() -> SyncReturn<String> {
 }
 
 pub fn close_channel() -> Result<()> {
-    ln_dlc::close_channel(false)
+    let runtime = ln_dlc::get_or_create_tokio_runtime()?;
+    runtime.block_on(async { ln_dlc::close_channel(false).await })
```

Comment:
This is not what I meant. I said that we should use the static runtime to spawn wherever we were running into this kind of panic: 

```
there is no reactor running, must be called from the context of a Tokio 1.x runtime
```

The whole idea was not to make `close_channel` and others `async`. Making things `async` unnecessarily is a risk because then we have to be careful not to add blocking code there.

Reply:
Have a look at https://github.com/get10101/10101/pull/1610